utils/techutils/techutils.go

```diff
@@ -400,13 +400,19 @@ func isExclude(path string, techData TechData) bool {
 func mapWorkingDirectoriesToTechnologies(workingDirectoryToIndicators map[string][]string, excludedTechAtWorkingDir map[string][]Technology, requestedTechs []Technology, requestedDescriptors map[Technology][]string) (technologiesDetected map[Technology]map[string][]string, err error) {
 	// Get the relevant technologies to check
 	technologies := requestedTechs
+	var techProvidedByUser bool
 	if len(technologies) == 0 {
 		technologies = GetAllTechnologiesList()
+	} else {
+		// If the project's technology was provided by the user, and isn't detected by us, we want to enable capturing the technology by its descriptor as well as by its indicators.
+		// In case we execute our auto-detection we want to avoid that since it may lead to collisions between package managers with the same descriptor (like Npm and Yarn)
+		techProvidedByUser = true
+		log.Debug(fmt.Sprintf("An install command was provided, so the technology will be identified based on the given command. Detected technologies: %s", technologies))
```

Comment:
tech provided not only by "install command" it could also be using a tech flag (not frog Frogbot). actually there is no mentioned in the code about the install command, only by requested tech.
fix the debug log and change the PR title to reflect the actual change